sbin/build.sh

```diff
@@ -224,8 +224,12 @@ configuringVersionStringParameter()
     addConfigureArg "--with-vendor-url=" "https://adoptopenjdk.net/"
     addConfigureArg "--with-vendor-name=" "AdoptOpenJDK"
     addConfigureArg "--with-vendor-bug-url=" "https://github.com/AdoptOpenJDK/openjdk-build/issues"
-    addConfigureArg "--with-vendor-vm-bug-url=" "https://github.com/eclipse/openj9/issues"
 
+    if [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_OPENJ9}" ]]; then
+      addConfigureArg "--with-vendor-vm-bug-url=" "https://github.com/eclipse/openj9/issues"
+    else
+      addConfigureArg "--with-vendor-vm-bug-url=" "https://github.com/AdoptOpenJDK/openjdk-build/issues"
```

Comment:
is this not already the default? https://github.com/AdoptOpenJDK/openjdk-jdk8u/blob/dev/jdk/src/share/native/java/lang/System.c#L114

Reply:
oh not on jdk9+